R/tm_front_page.R

```diff
@@ -89,8 +103,8 @@ tm_front_page <- function(label = "Front page",
     server = srv_front_page,
     ui = ui_front_page,
     ui_args = args,
-    server_args = list(tables = tables, show_metadata = show_metadata),
-    datanames = if (show_metadata) "all" else NULL
+    server_args = list(tables = tables),
+    datanames = unique(datanames)
```

Comment:
No need to do this. `teal::module` removes duplicated [see here](https://github.com/insightsengineering/teal/blob/e07fc33b7dd612141e589b6e41a598d9e37051dc/R/modules.R#L274)
```suggestion
    datanames = datanames
```